test/test-temp-directory.js

```diff
@@ -74,10 +74,7 @@ test('tempDirectory.create: bad path', skipIfWin32, async (t) => {
   try {
     await tempDirectory.create(badContext);
   } catch (e) {
-    t.ok(
-      e.message.includes(nullDevice),
-      `the message should include the path ${nullDevice}`
-    );
+    t.ok(e.message.includes('EEXIST'), `the message should include EEXIST`);
```

Comment:
While being on this, we might also switch to using `assert.rejects()`. Otherwise we do not verify that the catch clause is actually reached.

Reply:
Feel free to push to this PR, although if you do please use the built in tap asserts

https://node-tap.org/docs/api/asserts/#trejectspromise--fn-expectederror-message-extra

TBH the tests across the board could use a refactor so I'd prefer to save fixing this for a larger refactoring